source/localizable/smart_cart/_order_object.html.md.erb

```diff
@@ -84,11 +84,12 @@ Name                              | Type      | Description
 `line_items[_].total_price`     | Double  | Total item price in euros: `unit_price` * `quantity`
 `line_items[_].price_includes_vat` | Boolean  | Price includes VAT (could be `false` for [invoice](#invoice-details) with VAT exclusion)
 `line_items[_].ean`             | String  | EAN code of product (optional)
+`line_items[_].extra_info`      | String  | Customer's additional information for item (optional) [example](../webhook/#example-9-with-extrainfo-in-lineitems)
```

Comment:
Should we mention what does this attribute describe more specifically, i.e. mention color etc in some way?

Reply:
Also, since the example points to a webhook example from both the order and the webhook docs, I'd suggest to possibly change it to [webhook example] instead of [example], but feel free to keep whichever you prefer :)

Reply:
Sure, changed in 488cdf0b
Thanks